### ISSUE TITLE
Handle Debian in Shared oval files Platform tag

### ISSUE THREAD

**Issue by electrosenpai** [OpenSCAP/scap-security-guide#898]
Hi,

With integrated Debian on ssg and  we noticed that platform used in shared/oval some oval are using for example : 

```
<platform>multi_platform_rhel</platform>
```

Our main problem here is now we are working for Debian lot of test can be useful for us and we would like to use it by taking the shared folder. From my point of view there are 2 possibilities  :

We just add on every test :

```
<platform>debian8</platform>
```

Or the "good" solution should be create a 

```
<platform>multi_linux_platform</platform>
```

That contain every test usefull for all linux and a :

```
<platform>multi_platform_debian</platform>
```

For every platform base on debian (Ubuntu, Kali ...). 

We can obviously make it for you, just tell me want do you want.

Thank you, JB.

**Comment by electrosenpai** [comment]
Now we have a Debian repository we really need a help for this issue , thank's

**Comment by electrosenpai** [comment]
shoud be fix with #941
